roles/efk_opensearch_stack/defaults/main.yaml

```diff
@@ -13,6 +13,12 @@ elastic_opensearch_chart_version: "1.11.1"
 elastic_opensearch_values: {}
 elastic_opensearch_wait_for_deployments: true
 
+elastic_opensearch_dashboards_chart_name: "opensearch-dashboards"
+elastic_opensearch_dashboards_chart_version: "1.5.1"
+elastic_opensearch_dashboards_values: {}
+elastic_opensearch_dashboards_wait_for_deployments: true
+elastic_opensearch_dashboards_enable: false
```

Comment:
nit: have that one higher up?

Reply:
I've moved it to the top of that block.  It could be even higher, but then it's away from other Dashboards vars

Reply:
Top of the block is perfect, thanks